Авторизация. Захардкодили страницу авторизации пользователя

MikruykovaEkaterina opened the pull request:
**Реализована задача #3 Нарисовать окно авторизации**

**Изменения:**

- App.razor - добавлена обёртка <AuthRouter />
- AuthRouter.razor (новый) - управление логикой авторизации
- LoginModal.razor (новый) - форма авторизации с логином и паролем, кнопкой войти
- Program.cs - зарегистрирован AuthService
- AuthService.cs (новый) - сервис авторизации с захардкоженной логикой (всегда неавторизован)
- app.css - добавлены стили для окна авторизации

**Результат:**
Окно авторизации открывается на всех страницах сайта, так как пользователь всегда считается неавторизованным (захардкожено).
**Для проверки:**
Окно авторизации открывается на любой странице, например: https://localhost:7087 или https://localhost:7087/recipes/create

MikruykovaEkaterina committed:
добавлена обёртка AuthRouter

MikruykovaEkaterina committed:
Зарегистрирован AuthService

MikruykovaEkaterina committed:
сервис авторизации

MikruykovaEkaterina committed:
добавлены стили для окна авторизации

MikruykovaEkaterina committed:
Добавлен компонент формы авторизации

MikruykovaEkaterina committed:
управление логикой авторизации

MikruykovaEkaterina committed:
добавлена обёртка AuthRouter

Dezodemius commented on RecipeApp/RecipeApp/wwwroot/app.css:
```diff
@@ -392,4 +392,116 @@ article {
     color: #ff0300;
+}
+
+/* ����� ��� ���������� ���� ����������� */
```
а чё оно такое?

Dezodemius replied:
/* Ñòèëè äëÿ ìîäàëüíîãî îêíà àâòîðèçàöèè */

Dezodemius commented on RecipeApp/RecipeApp/wwwroot/app.css:
```diff
@@ -392,4 +392,116 @@ article {
+    text-transform: uppercase;
+}
+
+    .login-btn:hover {
```
лишний отступ?

Dezodemius replied:
Ну хоты выглядит логично. Окей

Dezodemius commented on RecipeApp/RecipeApp/wwwroot/app.css:
```diff
@@ -392,4 +392,116 @@ article {
+    margin-bottom: 1.5rem;
+}
+
+    .form-group label {
```
и здесь

Dezodemius replied:
Ну хоты выглядит логично. Окей

Dezodemius commented on RecipeApp/RecipeApp/wwwroot/app.css:
```diff
@@ -392,4 +392,116 @@ article {
+    margin-bottom: 2rem;
+}
+
+    .login-header h2 {
```
и здесь

Dezodemius replied:
Ну хоты выглядит логично. Окей

Dezodemius committed:
Merge branch 'dev' into DevelopmentAddAuthorization

Dezodemius committed:
Добавлен недостающий using

Последствия отказа от ImplicitUsings

MikruykovaEkaterina committed:
убрали код c# в отдельный файл

MikruykovaEkaterina committed:
Перенесли код c# из AuthRouter

MikruykovaEkaterina committed:
Перенесли код c# из LoginModal

MikruykovaEkaterina committed:
убрали код c# и добавили оператор await

MikruykovaEkaterina committed:
добавили документацию, регионы, исправили по комментария stylecop

Убрали явную инициализацию IsAuthenticated
Изменили тип события с Action? на EventHandler?
Добавили await Task.Delay(1) в асинхронный метод
Обновили вызовы события с Invoke() на Invoke(this, EventArgs.Empty)

MikruykovaEkaterina committed:
Merge branch 'DevelopmentAddAuthorization' of https://github.com/GladCom/RecipeApp into DevelopmentAddAuthorization

MikruykovaEkaterina committed:
добавили недостающий using

MikruykovaEkaterina committed:
Исправили по комментариям stylecop

MikruykovaEkaterina committed:
Исправили по комментариям stylecop

Dezodemius commented:
Ну шо там? Нужно ответить на комментарии. Я их закрою и поставлю аппрув

MikruykovaEkaterina committed:
Update app.css
